CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v3.32.0
```

Comment:
I think you also need to bump the version in `package.json`?

Reply:
Oh, yes 🤦‍♀️